Interoperability/python/src/qsharp/clients/iqsharp.py

```diff
@@ -65,7 +67,8 @@ def stop(self):
 
     def is_ready(self):
         try:
-            self.execute('%history', timeout=6)
+            result = self.execute('%version', timeout=6)
```

Comment:
```suggestion
            result = self.component_versions(timeout=6)
```

Otherwise, the versions are printed on the console.